allennlp/modules/stacked_elmo_lstm.py

```diff
@@ -0,0 +1,146 @@
+"""
+A stacked bidirectional LSTM with skip connections between layers.
+"""
+
+from typing import Optional, Tuple
+import torch
+from torch.nn.utils.rnn import PackedSequence, pad_packed_sequence
+from allennlp.modules.elmo_lstm_cell import ElmoLstmCell
+from allennlp.common.checks import ConfigurationError
+
+
+class ElmoLstm(torch.nn.Module):
```

Comment:
Same comment here as above regarding name.  This is a generic `StackedBidirectionalLSTMWithOutputProjecetionAndResidualConnection`